doc/iterations/2023.12/shapes/keep_node_aspect_ratio.adoc

```diff
@@ -0,0 +1,47 @@
+= (M) Keep node aspect ratio
+
+== Problem
+
+There is no possibility to configure a node to maintain its ratio in every case.
+
+== Key Result
+
+A specifier should be able to fix the default size on a node and specify if this ratio should be fixed.
+The diagram representation should apply these default values and keep the ratio in every layout situation.
+
+== Solution
+
+Add to the diagram DSL for `NodeDescription`:
```

Comment:
Shoudldn't these be on `NodeStyleDescription`?
In the presence of conditional style, I see no particular reason to assume that each style variant would have the same default size and `keepAspectRatio` value.

But then this opens the question: how are the new `default{Width,Height}Expression` different from the existing `{width,height}ComputationExpression` and how do they interact if we have both?